cns/NetworkContainerContract.go

```diff
@@ -60,6 +68,7 @@ type CreateNetworkContainerRequest struct {
 	LocalIPConfiguration       IPConfiguration
 	OrchestratorContext        json.RawMessage
 	IPConfiguration            IPConfiguration
+	SecondaryIPConfigs         map[string]IPSubnet // or struct {ipsubnet} //IPsubnet //uuid is key
```

Comment:
Currently not in use, do we need SecondaryIPConfigs to be a part of the CreateNetworkContainerRequest?

Reply:
Search for secondary IP's by NCID

Reply:
Add structure for IPSubnet